pkg/volume/util/types/types.go

```diff
@@ -99,8 +99,8 @@ func IsOperationFinishedError(err error) bool {
 // IsFilesystemMismatchError checks if mount failed because requested filesystem
 // on PVC and actual filesystem on disk did not match
 func IsFilesystemMismatchError(err error) bool {
-	mountError := mount.MountError{}
-	if errors.As(err, &mountError) && mountError.Type == mount.FilesystemMismatch {
+	mountError := &mount.MountError{}
+	if errors.As(err, mountError) && mountError.Type == mount.FilesystemMismatch {
```

Comment:
is this change unrelated?

Reply:
It is unrelated. Updating a dependency caused some tests to fail and this was the fix